components/mailbox/src/Mailbox.svelte

```diff
@@ -523,8 +530,23 @@
 
     [role="toolbar"] {
       @include barStyle;
+      border-bottom: 0.5px solid
+        var(--mailbox-border-color, var(--grey-lighter));
       padding: $spacing-s $spacing-m;
       gap: $spacing-m;
+
+      .thread-checkbox {
+        display: flex;
+      }
```

Comment:
I don't think I agree with our current treatment of this checkbox. When I see it checked, I assume all emails (in this view, or in the mailbox) are also checked. When it's unchecked, i assume all emails are unchecked.

As it stands, it will become checked if one checks a single email in the list. IMO it should mimic the behaviour of, for example, https://dashboard.nylas.com/applications/app/webhooks/new or if possible, gmail (gmail's treatment is a little more advanced -- it has a "some selected" horizontal line state)

Reply:
Is it not the same as gmail currently?
 - If nothing is checked, this will check all boxes on the current page
 - If at least one checkbox is checked, it will uncheck all boxes on the current page
 - Checkbox status is maintained as pages are changed

Reply:
also just noticed that our webhooks page breaks back navigation - you can't go back using the browser from that link